Player/Player.cs

```diff
@@ -237,6 +237,27 @@ public void InventoryDown()
 		}
 	}
 
+	public void InventorySlotSelect(int Slot)
+	{
+		InventorySlot = Slot;
+
+		if(HUDInstance != null)
```

Comment:
The `HUDInstance` field is filled during the `Player` constructor. Did you an issue which caused the inclusion of this null check?

Reply:
No, I saw this check was present in InventoryUp/Down so I put it too.

Reply:
Ahhh, that is odd. They are probably left over from when we always had a background copy of `Player` under `Game.PossessedPlayer` when they were required to prevent a crash when scrolling on the main menu. I will remove the checks in InventoryUp/Down after this is merged. Would you like to remove the check from this PR or would you like for me to do that after merging?